code/API_definitions/kyc-match.yaml

```
@@ -0,0 +1,573 @@
+                  streetNumber: 4
+                  postalCode: 1028460
+                  region: Tokyo
+                  locality: ZZZZ
+                  country: Japan
+                  houseNumberExtension: VVVV
+                  birthdate: '1978-08-22'
+                  email: abc@example.com
+                  gender: male
+
+      responses:
+        '200':
+          description: OK
+          headers:
+            x-correlator:
+              $ref: '#/components/headers/x-correlator'
+
+          content:
+            application/json;charset=UTF-8:
+              schema:
+                $ref: '#/components/schemas/KYC_MatchResponse'
+              examples:
+                KYC_Match200Example:
+                  value:
+                    phoneNumberMatch: 'true'
+                    idDocumentMatch: 'true'
+                    nameMatch: 'true'
+                    giventNameMatch: 'not_available'
+                    familyNameMatch: 'not_available'
+                    nameKanaHankakuMatch: 'true'
+                    nameKanaZenkakuMatch: 'false'
+                    middleNamesMatch: 'true'
+                    familyNameAtBirthMatch: 'false'
+                    addressMatch: 'true'
+                    streetNameMatch: 'true'
+                    streetNumberMatch: 'true'
+                    postalCodeMatch: 'true'
+                    regionMatch: 'true'
+                    localityMatch: 'not_available'
+                    countryMatch: 'true'
+                    houseNumberExtensionMatch: 'not_available'
+                    birthdateMatch: 'false'
+                    emailMatch: 'false'
+                    genderMatch: 'false'
+
+        '400':
+          $ref: '#/components/responses/Generic400'
+
+        '401':
+          $ref: '#/components/responses/Generic401'
+
+        '403':
+          $ref: '#/components/responses/Generic403'
+
+        '404':
+          $ref: '#/components/responses/Generic404'
+
+        '500':
+          $ref: '#/components/responses/Generic500'
+
+        '503':
+          $ref: '#/components/responses/Generic503'
+
+        '504':
+          $ref: '#/components/responses/Generic504'
+
+components:
+  securitySchemes:
+    openId:
+      type: openIdConnect
+      openIdConnectUrl: https://example.com/.well-known/openid-configuration
+
+
+  headers:
+    x-correlator:
+      description: Correlation id for the different services
+      schema:
+        type: string
+   
+  parameters:
+    x-correlator:
+      name: x-correlator
+      in: header
+      description: Correlation id for the different services
+      schema:
+        type: string
+        
+  schemas:
+    KYC_MatchRequestBody:
+      type: object
+      description: Payload to validate the customer data.
+
+      properties:
+        phoneNumber:
+          type: string
+          description: A phone number belonging to the user in E.164 format (starting with country code). Optionally prefixed with '+'.

```

An this one just to align with the commonalities Glossary

```suggestion
          description: A public identifier addressing a telephone subscription. In mobile networks it corresponds to the MSISDN (Mobile Station International Subscriber Directory Number). In order to be globally unique it has to be formatted in international format, according to E.164 standard, optionally prefixed with '+'.

```

---

> An this one just to align with the commonalities Glossary

Thank you very much, but never ending.  Could we make this the last change, please?  GSMA is waiting.  We can do corrections later.